docs/rules/prefer-reflect.md

```diff
@@ -18,20 +18,22 @@ The prefer-reflect rule will flag usage of any older method, suggesting to inste
 ### Exceptions
 
 ```
-"prefer-reflect": [<enabled>, { exceptions: [<...exceptions>] }]
+"prefer-reflect": [<enabled>, { "exceptions": [<...exceptions>] }]
 ```
 
 The `exceptions` option allows you to pass an array of methods names you'd like to continue to use in the old style.
 
-For example if you wish to use all Reflect methods, except for `Function.prototype.apply` then your config would look like `prefer-reflect: [2, { exceptions: ["apply"] }]`.
+For example if you wish to use all Reflect methods, except for `Function.prototype.apply` then your config would look like `prefer-reflect: [2, { "exceptions": ["apply"] }]`.
 
-If you want to use Reflect methods, but keep using the `delete` keyword, then your config would look like `prefer-reflect: [2, { exceptions: ["delete"] }]`.
+If you want to use Reflect methods, but keep using the `delete` keyword, then your config would look like `prefer-reflect: [2, { "exceptions": ["delete"] }]`.
 
-These can be combined as much as you like. To make all methods exceptions (thereby rendering this rule useless), use `prefer-reflect: [2, { exceptions: ["apply", "call", "defineProperty", "getOwnPropertyDescriptor", "getPrototypeOf", "setPrototypeOf", "isExtensible", "getOwnPropertyNames", "preventExtensions", "delete"] }]`
+These can be combined as much as you like. To make all methods exceptions (thereby rendering this rule useless), use `prefer-reflect: [2, { "exceptions": ["apply", "call", "defineProperty", "getOwnPropertyDescriptor", "getPrototypeOf", "setPrototypeOf", "isExtensible", "getOwnPropertyNames", "preventExtensions", "delete"] }]`
 
-### Reflect.apply (Function.prototype.apply/Function.prototype.call)
+### Reflect.apply
 
-The following patterns are considered problems:
+> Deprecates `Function.prototype.apply()` and `Function.prototype.call()`
```

Comment:
I don't think `>` (block quote) is the correct treatment for this or any other place in this PR. Just use regular text.